tests/sqllogictests/suites/base/09_fuse_engine/09_0048_issue_17936.sql

```diff
@@ -0,0 +1,44 @@
+statement ok
+CREATE OR REPLACE SEQUENCE test_seq;
+
+statement ok
```

Comment:
we can add another column of uint64 not null，which cover the code path where no cast needed，my lasr pr miss this too. 

by the way, nextval is uint64, int may be not big enough?  all cases in docs also use int: https://docs.databend.com/sql/sql-functions/sequence-functions/nextval
https://docs.databend.com/sql/sql-commands/ddl/table/ddl-create-table#column-default-values